acond/src/utils.rs

```diff
@@ -701,43 +702,24 @@ pub fn generate_cid() -> Result<u32> {
     }
 
     let overflow_uid = contents.parse::<u32>()?;
-    let cid = CONTAINER_SERIES.fetch_add(1, Ordering::Relaxed) as u32;
+    let cid = CONTAINER_SERIES.fetch_add(1, Ordering::Relaxed);
     if cid != overflow_uid {
         return Ok(cid);
     }
 
-    Ok(CONTAINER_SERIES.fetch_add(1, Ordering::Relaxed) as u32)
+    Ok(CONTAINER_SERIES.fetch_add(1, Ordering::Relaxed))
 }
 
-pub fn parse_content_range(header: &str) -> Option<u64> {
-    let parts: Vec<&str> = header.split("bytes").collect();
-    if parts.len() != 2 {
-        return None;
+pub fn extract_start(content_range: &ContentRange) -> Option<u64> {
```

Comment:
This func seems used only once. Why not inlining it?

Reply:
Fixed